ASN1Decoder/ASN1Object.swift

```diff
@@ -33,9 +33,9 @@ public class ASN1Object: CustomStringConvertible {
 
     public var identifier: ASN1Identifier?
 
-    var sub: [ASN1Object]?
+    public internal(set) var sub: [ASN1Object]?
 
-    weak var parent: ASN1Object?
+    public internal(set) weak var parent: ASN1Object?
```

Comment:
Those changes come from https://github.com/filom/ASN1Decoder/pull/32 on which this PR is based on.